test/common.jl

```diff
@@ -0,0 +1,21 @@
+using TaylorIntegration, Base.Test
+
+@testset "Test integration of ODE with numbers in common interface" begin
+    f(t,u) = u
+    u0 = 0.5
+    tspan = (0.0,1.0)
+    prob = ODEProblem(f,u0,tspan)
+    sol = solve(prob,TaylorMethod(50))
+
+    @test sol[end] - 0.5exp(1) < 1e-12
```

Comment:
If the default value for `abstol` is set to `1e-20`, this test can check the equality!

Reply:
I just checked that equality also holds with `abstol = 1e-6`.